frontends/precinct-scanner/src/app_unhappy_paths.test.tsx

```diff
@@ -389,6 +389,7 @@ test('removing card during calibration', async () => {
   const logger = fakeLogger();
   const card = new MemoryCard();
   const hardware = MemoryHardware.buildStandard();
+  hardware.setPrinterConnected(false);
```

Comment:
The new "require prints be asserted against" requirement of the testing utility led me to notice that we were printing a report in this test where it is not relevant. 

We should probably stop using `buildStandard` for all these tests when that doesn't represent the hardware setup of the actual scanner.